core/object/script_language.cpp

```diff
@@ -34,6 +34,7 @@
 #include "core/core_string_names.h"
 #include "core/debugger/engine_debugger.h"
 #include "core/debugger/script_debugger.h"
+#include "editor/editor_node.h"
```

Comment:
Core should not depend on Editor.
The info you're getting from it is likely in ClassDB.

Reply:
OK, I think I can implement it without the editor too.